onnx/gen_proto.py

```diff
@@ -42,16 +42,27 @@ def process_ifs(lines, onnx_ml):
                 yield line
 
 
+IMPORT_REGEX = re.compile(r'(\s*)import\s*"([^"]*)\.proto";\s*$')
 PACKAGE_NAME_REGEX = re.compile(r'\{PACKAGE_NAME\}')
+ML_REGEX = re.compile(r'(.*)\-ml')
 
 def process_package_name(lines, package_name):
     for line in lines:
-        yield PACKAGE_NAME_REGEX.sub(package_name, line)
+        m = IMPORT_REGEX.match(line)
+        if m:
+            include_name = m.group(2)
+            ml = ML_REGEX.match(include_name)
+            if ml:
```

Comment:
nit: maybe use onnx-ml_PACKAGENAME.proto to avoid having different processing logic here?